rules.neon

```diff
@@ -64,12 +64,16 @@ conditionalTags:
 		phpstan.rules.rule: %strictRules.booleansInConditions%
 	PHPStan\Rules\BooleansInConditions\BooleanInBooleanOrRule:
 		phpstan.rules.rule: %strictRules.booleansInConditions%
+	PHPStan\Rules\BooleansInConditions\BooleanInDoWhileConditionRule:
+		phpstan.rules.rule: %strictRules.booleansInConditions%
```

Comment:
This is a new rule. It has to be part of bleeding edge. See how this was done in 1.x: https://github.com/phpstan/phpstan-strict-rules/blob/b564ca479e7e735f750aaac4935af965572a7845/rules.neon#L20

This also means this needs a new config parameter. Something like booleansInLoopConditions?

Reply:
Sure, should be ok now